pom.xml

```diff
@@ -350,6 +350,16 @@
                     </vocabularies>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.2</version>
+                <configuration>
+                    <includes>
+                        <include>**/*.java</include>

```

Comment:
Is this different from the default settings?

Reply:
Nope: https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html

"Test" or "TestCase" must be in the names by default... so tests like "List_POST.java" are not discovered.